lib/swarm/distribution/static_quorum_ring.ex

```diff
@@ -55,8 +55,13 @@ defmodule Swarm.Distribution.StaticQuorumRing do
   defstruct [:static_quorum_size, :ring]
 
   def create do
+    static_quorum_size =
+      Application.get_env(:swarm, :static_quorum_size, 2)
```

Comment:
Won't this break if the env is not set because it's being set to an integer by default, then calling `Integer.parse()` on an integer rather than a string? 

Perhaps this might work nicer: 

```
Application.get_env(:swarm, :static_quorum_size, "2") 
   |> Integer.parse() 
|> elem(0)
```

Reply:
true that

Reply:
also if the variable is set by hand in config as an integer it will also break

Reply:
I can also create some helper function which will do that by pattern match the variable, however I didn't want to interfere too much in other library code